binfmt/libelf/libelf_bind.c

```diff
@@ -326,8 +367,181 @@ static int elf_relocate(FAR struct elf_loadinfo_s *loadinfo, int relidx,
 static int elf_relocateadd(FAR struct elf_loadinfo_s *loadinfo, int relidx,
```

Comment:
Let's enhance modlib_relocateadd too, maybe it's better to split RELA to another patch.

Reply:
> Let's enhance modlib_relocateadd too, 
> maybe it's better to split RELA to another patch.

@xiaoxiang781216 Thanks for the comments. I'll consider them later.